azure-pipelines.snapshots.yml

```diff
@@ -0,0 +1,47 @@
+pr: none
+trigger: none
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+variables:
+  - group: 'Github and NPM secrets'
+
+steps:
+  - checkout: self
+    persistCredentials: true
+
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
+
+  - script: |
+      yarn
+    displayName: yarn
+
+  - script: |
+      yarn build --min
+    displayName: build (no test or lint)
+
+  - script: |
+      yarn update-snapshots
+    displayName: update snapshots
+
+  - script: |
+      git config --global user.email "fabrictactical@microsoft.com"
+      git config --global user.name "Fabric Tactical"
+    displayName: Set up git config user name and email
+
+  - script: |
+      if [ -n '$(git status --porcelain | grep -E "(\.snap|\.shot)")' ]; then
+        noauthRemote=$(remote)
+        git remote add target_remote ${noauthRemote/\/github.com/\/$(githubUser):$(githubPAT)@github.com}
```

Comment:
Hmm should the recovery build be in classic editor? Otherwise it seems people can modify this build in a fork build and leak secrets?

Reply:
That's a pretty good point... pretty easily converted I believe

Reply:
removed!